repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -135,13 +139,26 @@ def process_events(events):
         api.produce(RepositoriesSetupTasks(to_enable=list(to_enable_repos)))
 
 
+def verify_pes_data(path):
```

Comment:
am I missing something or is this function not used anywhere,
I guess you wanted to use it in parse_file?